app/views/devise/registrations/new.html.haml

```diff
@@ -1,46 +1,6 @@
-
-= content_for :page_title, flush: true do
-  = t('.page_title')
-
-= render partial: 'layouts/header', locals: { page_heading: t('.page_heading') }
-
-= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-  - if resource.errors.any?
-    .validation-summary
-      %h2.heading-small= "This #{resource_name} has #{pluralize(resource.errors.count, 'error')}"
-      %ul
-        - resource.errors.each do |attribute, message|
-          %li
-            = "#{attribute.to_s.humanize} #{message}"
-
-  .form-group
-    = render 'terms_and_conditions'
-
-    = label_tag :terms_and_conditions_acceptance, required: true, class: 'form-label-bold' do
-      = check_box_tag :terms_and_conditions_acceptance
-      Accept terms and conditions
-
-  .form-group
-    = f.label :first_name, class: 'form-label'
-    = f.input_field :first_name, class: 'form-control'
-
-  .form-group
-    = f.label :last_name, class: 'form-label'
-    = f.input_field :last_name, class: 'form-control'
-
-  .form-group
-    = f.label :email, required: true, class: 'form-label'
-    = f.input_field :email, class: 'form-control'
-
-  .form-group
-    = f.label :password, required: true, class: 'form-label'
-    = f.input_field :password, class: 'form-control'
-
-  .form-group
-    = f.label :password_confirmation, required: true, class: 'form-label'
-    = f.input_field :password_confirmation, class: 'form-control'
-
-  .form-group
-    = govuk_button('Sign up')
-
-= render 'devise/shared/links'
+/ NOT USED: users should not be able to sign up themselves.
+/ Default devise view can be regenerated if needed using
+/ `rails generate devise:views -v registrations`
+/ but if you do so ensure the users/registrations_controller.rb
+/ prevents sign up by individuals.
+/
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3698/files#r562072907)